cal/cal/models.py

```diff
@@ -664,8 +666,8 @@ def query(self, calendar_ids=None, start=None, end=None):
 
         if start:
             events_qs = events_qs.filter(end__gt=start)
-        if end:
```

Comment:
Our tag list component relies on being able to set an end time, for the daily/weekly/monthly views. If you explore to a date in the future, tags would thus be inaccurate.